cs251tk/student/markdownify/markdownify.py

```diff
@@ -1,27 +1,30 @@
+"""Given a spec, assuming we're in the homework folder, run the spec against the folder"""
+
 import os
 from collections import OrderedDict
 from .process_file import process_file
 from .find_warnings import find_warnings
 
 
 def markdownify(spec_id, username, spec, basedir, debug):
+    """Run a spec against the current folder"""
     try:
         cwd = os.getcwd()
         results = {
             'spec': spec_id,
             'student': username,
-            'warnings': {},
+            'warnings': find_warnings(),
             'files': OrderedDict(),
         }
 
+        # prepare the current folder
         inputs = spec.get('inputs', [])
         supporting = os.path.join(basedir, 'data', 'supporting')
+        # write the supporting files into the folder
         for filename in inputs:
-            in_path = os.path.join(supporting, spec_id, filename)
-            out_path = os.path.join(cwd, filename)
-            with open(in_path, 'rb') as infile:
+            with open(os.path.join(supporting, spec_id, filename), 'rb') as infile:
                 contents = infile.read()
-            with open(out_path, 'wb') as outfile:
+            with open(os.path.join(cwd, filename), 'wb') as outfile:
```

Comment:
Sounds good. Is this not more elegant?  I suppose having these as variables _would_ be nice.